src/pages/CreateEventPage/styled.ts

```diff
@@ -0,0 +1,17 @@
+import styled from 'styled-components';
+import { ThemeProps } from '../../hooks/theme';
+
+export const CreateEventWrapper = styled.div`
+  margin: 0 2.5vw;
+`;
+
+export const Form = styled.form`
+  margin-bottom: 20px;
+  font-weight: bold;
+  width: ${({ theme }: ThemeProps) => theme.typography.scaleSpacing(16)}px;
+  height: ${({ theme }: ThemeProps) => theme.typography.scaleSpacing(17)}px;
+
+  input, textarea, select {
+    display: block;
+  }
```

Comment:
Same as the other comment regarding implicit styling. If I were to write a new field component that made use of the `input` element, there's no way I would know that my element would implicitly get the `display: block` rule set without debugging and finding this file.

Reply:
Is there a way to have webpack/react build system automatically scope these, something like what you can do with SASS?

Reply:
Are you thinking of the scoping that you get with CSS Modules? Because I don’t think Sass itself has any scoping feature.

Assuming that you are indeed thinking about scoping as how it’s done with CSS Modules, then this same issue still persists - I believe they use the same scoping technique behind the scenes. The scoping only encapsulates each «CSS Module» from each other, but doesn’t neccessary stop an ancestor element’s styling from affecting its descendents. The only way to avoid that is to use class- or id-selectors, which are scoped, I think.